docs/guides/qunasys-quri-chemistry.ipynb

```diff
@@ -354,32 +514,38 @@
     "\n",
     "job = function.run(\n",
     "    method=\"QSCI\",\n",
-    "    parameters=qsci_double_exc_json,\n",
-    "    instance=\"INSTANCE\",\n",
-    "    backend=\"ibm_strasbourg\",\n",
+    "    molecule=molecule,\n",
+    "    circuit_options=qsci_double_exc_json,\n",
+    "    qsci_setting=qsci_setting,\n",
+    "    mitigation_setting=mitigation_setting,\n",
+    "    backend_name=\"ibm_strasbourg\",\n",
     ")\n",
     "\n",
     "while True:\n",
-    "    print(datetime.datetime.now(), job.status())\n",
-    "    if job.status() in [\"ERROR\", \"DONE\"]:\n",
-    "        break\n",
-    "    time.sleep(10)\n",
+    "    try:\n",
+    "        print(datetime.datetime.now(), job.status())\n",
+    "        time.sleep(10)\n",
+    "        if job.status() in [\"ERROR\", \"DONE\"]:\n",
+    "            break\n",
+    "    except Exception:\n",
```

Comment:
Do you know what error you were catching? This is a bad practice to use such a general except.